maestro-ios-driver/src/main/kotlin/util/CommandLineUtils.kt

```diff
@@ -24,7 +24,7 @@ object CommandLineUtils {
         } else {
             ProcessBuilder(*parts.toTypedArray())
                 .redirectOutput(nullFile)
-                .redirectError(nullFile)
+                .redirectError(ProcessBuilder.Redirect.PIPE)
```

Comment:
The errors were not even redirecting to the process due to which it just throws illegal state exception without any meaningful message